docs/pages/base/api/table-pagination-unstyled.json

```diff
@@ -39,35 +38,7 @@
     }
   },
   "name": "TablePaginationUnstyled",
-  "styles": {
-    "classes": [
-      "root",
-      "toolbar",
-      "spacer",
-      "selectLabel",
-      "selectRoot",
-      "select",
-      "selectIcon",
-      "input",
-      "menuItem",
-      "displayedRows",
-      "actions"
-    ],
-    "globalClasses": {
-      "root": "MuiTablePagination-root",
-      "toolbar": "MuiTablePagination-toolbar",
-      "spacer": "MuiTablePagination-spacer",
-      "selectLabel": "MuiTablePagination-selectLabel",
-      "selectRoot": "MuiTablePagination-selectRoot",
-      "select": "MuiTablePagination-select",
-      "selectIcon": "MuiTablePagination-selectIcon",
-      "input": "MuiTablePagination-input",
-      "menuItem": "MuiTablePagination-menuItem",
-      "displayedRows": "MuiTablePagination-displayedRows",
-      "actions": "MuiTablePagination-actions"
-    },
-    "name": null
-  },
+  "styles": { "classes": [], "globalClasses": {}, "name": null },
```

Comment:
Do we need to find a new way of how to document the global classes for the unstyled components?

Reply:
Yes, since the unstyled components don't have the `classes` prop, we'll have to figure out something else. Perhaps the solution will appear when we allow customizing/disabling the default classes.